docs/infrastructure/fern/GENERATING.md

```diff
@@ -0,0 +1,19 @@
+# Generating
+
+## Step 1: Cloud & build Fern
+
+```sh
+gh repo clone rivet-gg/fern
```

Comment:
In my version, I said to check out `max/remove-headers`. Is that now included in trunk?